packages/algob/src/lib/tx.ts

```diff
@@ -266,16 +268,68 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object(s) and returns raw signed transaction
+ * Note: Sign transaction is used to sign single transaction and `signTransactionObject` takes
+ * SDK transaction object, signer and signs it.
+ * @param txn Transaction object(s)
+ * @param signer signer
```

Comment:
please add more documentation about the behavior of signer when it's a single object or an array. Add that if it's an array, then the length must be the same as `txns` length.

Reply:
ok